webapp/Connector/src/app/store/Study.js

```diff
@@ -94,6 +99,11 @@ Ext.define('Connector.app.store.Study', {
         this._onLoadComplete();
     },
 
+    onLoadLearnAssays : function(learnAssayData) {
+        this.learnAssayData = learnAssayData.rows;
```

Comment:
not sure why learnAssayData is needed since assayData already contains assay information. It's a left join from studyassay to assay, so should already be able to tell if a assay page exist.

Reply:
what about the data that is in Assay but not in StudyAssay (our initial implementation), in which case assayData won't have the info we need.

Reply:
I don't think that's a legit scenario. A study page will only list assays defined in studyassay relationship. StudyDocument is to provide detailed info for the relationship, but it doesn't define the relationship.